### ISSUE TITLE
Accessibility 

### ISSUE THREAD

**Issue by matthall8** [fac-15/I-Choose-Who-#22]
Although accessibility was not the core focus of the week, it should be an important consideration each and every time a website is built. The site scores pretty low from an accessibility perspective (22/100). 

![image](https://user-images.githubusercontent.com/9437789/48939267-9caef780-ef4e-11e8-9785-23151528178f.png)

The following should be fixed to make it more accessible:
![image](https://user-images.githubusercontent.com/9437789/48939295-bcdeb680-ef4e-11e8-9c76-3d84a13c2e3e.png)
